02-oop-inheritance/javaScript/refactored_JS/downloableMovie v0.1.js

```diff
@@ -0,0 +1,94 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function ( paramName, paramValue ) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
+  this.observer = new MovieObserver();
+};//end movie constructor
+
+Movie.prototype = {
+  
+  get: function (atribute) {
+     return this.attributes[atribute];
```

Comment:
I thought this model as this: 
movie.set('name', 'movieName');
movie.attributes.name 

the last sentence will return the value 'movieName', by giving the key 'name' of the hashTable